fusor-ember-cli/app/templates/rhev.hbs

```diff
@@ -4,15 +4,15 @@
         <a>2A. Setup Type</a>
       {{/link-to}}
 
+      {{#unless isSelfHost}}
       {{#link-to 'engine' tagName='li' disabled=disableTabRhevEngine}}
-        <a>2B. {{engineTabName}}</a>
+        <a>2B. Engine</a>
       {{/link-to}}
+      {{/unless}}
 
-      {{#unless isSelfHost}}
       {{#link-to 'hypervisor' tagName='li' disabled=disableTabRhevHypervisors}}
-        <a>2C. Hypervisors</a>
+        <a>2C. {{hypervisorTabName }}</a>
```

Comment:
I'm seeing this, so obviously the `hypervisorTabName` is not working correctly.

2B. Engine
2C. Engine